controller/volume_controller.go

```diff
@@ -2853,7 +2853,21 @@ func (vc *VolumeController) checkForAutoDetachment(v *longhorn.Volume, e *longho
 	}
 
 	if v.Status.ExpansionRequired {
-		return nil
+		if v.Status.CurrentNodeID != "" {
+			_, err := vc.ds.GetNodeRO(v.Status.CurrentNodeID)
+			if err == nil {
+				log.Debugf("Expansion of volume %v is required", v.Name)
```

Comment:
Let's remove this part because it would be very verbose and often, but just log the case needed to do auto detachment.

Reply:
Updated. Thank you.